atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchManagerImpl.java

```diff
@@ -36,40 +43,54 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public final class LockWatchManagerImpl extends LockWatchManager implements AutoCloseable {
+public final class LockWatchManagerImpl extends LockWatchManagerInternal {
 
     private static final Logger log = LoggerFactory.getLogger(LockWatchManagerImpl.class);
 
     private final Set<LockWatchReferences.LockWatchReference> referencesFromSchema;
     private final Set<LockWatchReferences.LockWatchReference> lockWatchReferences = ConcurrentHashMap.newKeySet();
-    private final LockWatchEventCache lockWatchEventCache;
+    private final LockWatchCache lockWatchCache;
+    private final LockWatchValueScopingCache valueScopingCache;
```

Comment:
There should be a method on LWC to get LWVSC, so no need to keep as a field.

Reply:
There cannot be one, for the same reason that LWVSC exists in the first place

Reply:
ah damn

Reply:
(Might be worth noting that or commenting that we do _not_ need to update the transaction state in both LWC and LWVSC).